web/src/App.js

```diff
@@ -49,16 +51,28 @@ function App() {
   }, []);
 
   const menuBarStyle = {
-    // primaryColor: "#f7c873",
-    // backgroundColor: "#f7c873",
-    //color: "#bd8e02",
     color: '#434343',
     textColor: '#434343',
     backgroundColor: "#001529"
   };
 
   const [isShown, setIsShown] = useState(false);
 
+  const dropdown1 = (
```

Comment:
I think it would be better to define this component outside App.js and import it here. What do you think @Dogacel  ?

Reply:
Yes, this file gets bigger and bigger. It is not necessary right now but if we are going to add more to the header, we should move the header component outside and just leave routes/structure logic inside App.js.